src/main/resources/org/jvnet/hudson/plugins/thinbackup/ThinBackupMgmtLink/index.jelly

```diff
@@ -18,13 +18,15 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
   xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <l:layout permission="${app.ADMINISTER}" norefresh="true">
+  <l:layout permission="${app.ADMINISTER}" norefresh="true" title="${it.displayName}">
```

Comment:
remove the `norefresh` attribute, it is deprecated and has no effect, same in restoreOptions.jelly, there the title attribute is missing